Testscripts/Linux/LIS-Mount-CD.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+########################################################################
+
+###############################################################
+#
+# Description:
+#     This script was created to automate the testing of a Linux
+#     Integration services. This script detects the CDROM
+#     and performs read operations .
+#
+################################################################
+
+dos2unix utils.sh
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    exit 0
+}
+
+#
+# Source constants file and initialize most common variables
+#
+UtilsInit
+
+GetGuestGeneration
+if [ $os_GENERATION -eq 1 ] && [ $HotAdd = "True" ]; then
+    SetTestStateSkipped
+    exit 0
+fi
+
+#
+# Check if the CDROM module is loaded
+#
+if [[ -x $(which lsb_release 2>/dev/null) ]]; then
+    os_VENDOR=$(lsb_release -i -s)
+fi
+if [ $os_VENDOR != "Ubuntu" ] && [ $os_VENDOR != "Debian" ]; then
+    CD=`lsmod | grep 'ata_piix\|isofs'`
+    if [[ $CD != "" ]] ; then
+        module=`echo $CD | cut -d ' ' -f1`
+        LogMsg "${module} module is present."
+    else
+        LogMsg "ata_piix module is not present in VM"
+        LogMsg "Loading ata_piix module "
+        insmod /lib/modules/`uname -r`/kernel/drivers/ata/ata_piix.ko
+        sts=$?
+        if [ 0 -ne ${sts} ]; then
+            LogMsg "Unable to load ata_piix module"
```

Comment:
put these 2 LogMsg into a single line, and make the updatesummary msg with the same text inside

Reply:
^^ Actually we need to get rid of UpdateSummary lines since it's not used anymore in lisav2